src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
```

Comment:
normal is default value, you can remove this style

Reply:
understood